settings/global.mdx

```diff
@@ -249,7 +249,7 @@ Example:
   ```
   <Expandable title="FooterSocials">
     <ResponseField name="[key]" type="string">
-    One of the following values `website`, `facebook`, `twitter`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
+    One of the following values `website`, `facebook`, `twitter`, `x`, `youtube`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
```

Comment:
Let's also removed `twitter` here. It can be allowed for backwards compatibility, but not suggested